src/admm.jl

```diff
@@ -0,0 +1,139 @@
+"""
+        admm(
+            m::AbstractBlockNLPModel;
+            kwargs... 
+        )
+Solves a `BlockNLPModel` with the ADMM algorithm.
+
+# Arguments
+
+- `m::AbstractBlockNLPModel`: identifier for the `BlockNLPModel` 
+- `kwargs...`: options for the solver
+"""
+function admm(
+    m::AbstractBlockNLPModel;
```

Comment:
I would suggest using a different name than `m` (single letters are not that explicit...)